index.html

```diff
@@ -1 +1,26 @@
-
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width,initial-scale=1" />
+		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
+		<style>
+			
```

Comment:
Why did you add the `style` if you don't even use CSS?

Reply:
We might use some and it's nice to have the tags.